src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/validation/ParticipationValidatorTest.java

```diff
@@ -38,48 +43,73 @@ void setup() {
 
   @Test
   void player_join_game_valid_game_state() {
-    PARTICIPATION_1.setParticipationNumber(0);
+    PARTICIPATION.setParticipationNumber(0);
```

Comment:
Is there a reason for sequence number 0,1,3? Why not 1,2,3 or 0,1,2?

Reply:
Nein, ist random, dürfte ich wahrscheinlich auch weglassen.